Modules/Core/Common/src/itkPoolMultiThreader.cxx

```diff
@@ -190,10 +192,8 @@ PoolMultiThreader ::ParallelizeArray(SizeValueType             firstIndex,
     }
     itkAssertOrThrowMacro(workUnit <= m_NumberOfWorkUnits, "Number of work units was somehow miscounted!");
 
-    ExceptionHandlingPrologue;
-
     // execute this thread's share
-    WrapExceptionHandling(lambda(firstIndex, firstIndex + chunkSize));
+    ExceptionHandler exceptionHandler([lambda, firstIndex, chunkSize] { lambda(firstIndex, firstIndex + chunkSize); });
```

Comment:
Hans thought that variable name `lambda` is not descriptive enough, now is a good time to change it. Not necessary, though.

Reply:
Honestly it's actually slightly worse now because I placed the variable named `lambda` inside a newly introduced _unnamed_ lambda! I could entirely get rid of the named `lambda` and just use two small unnamed lambda's. Would you like that? @hjmjohnson

Reply:
The reason for introducing a named lambda was to reduce code duplication. It being invoked from another, unnamed lambda is not a problem.